sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/sources/RateStreamProviderSuite.scala

```diff
@@ -305,7 +305,7 @@ class RateStreamProviderSuite extends StreamTest {
         .load()
     }
     assert(exception.getMessage.contains(
-      "rate source does not support user-specified schema"))
+      "RateStreamProvider source does not support user-specified schema"))
```

Comment:
This is an unrelated change and should be done in a separate commit.

Reply:
This is because I hesitate to move `DataSourceRegister` to catalyst, and then DS v2 is not able to get `DataSourceRegister#shorName`.